sim/mcuboot-sys/csupport/run.c

```diff
@@ -238,8 +238,15 @@ int flash_area_get_sectors(int fa_id, uint32_t *count,
 void sim_assert(int x, const char *assertion, const char *file, unsigned int line, const char *function)
 {
     if (!(x)) {
-        c_catch_asserts
-            ? c_asserts++
-            : __assert_fail(assertion, file, line, function);
+        if (c_catch_asserts) {
```

Comment:
Should we fold this into the patch that adds sim_assert? Right now, we have a bisection regression on MacOS.